lib/functions.inc.php

```diff
@@ -134,13 +134,150 @@ function make_md4_password($password) {
     return $hash;
 }
 
+/**
+ * Generate a hash with the given algorithm and options.
+ * @param string $password the password to hash
+ * @param string $hash the algorithm to be used
+ * @param array $hash_options additional options to be used by the hashing algorithm
+ * @return string
+ */
+function make_password($password, $hash, $hash_options): string {
```

Comment:
Typically, I think that these functions should go into https://github.com/ltb-project/ltb-ldap

@coudot what is your opinion on this?

Reply:
See https://github.com/ltb-project/ltb-ldap/pull/7